salesforce/utam-js-recipes
# chore(yarn): update to modern yarn

Updates the project to use to modern yarn.

You may need to take a step to make corepack override Volta's shims locally if you have issues. See the note in the readme below.

How to test:
Check out this branch. Make sure you have node >= 16 and try running just `yarn`. If that fails to upgrade your machine to yarn 4 automatically, run `corepack use yarn@4.0.2`, then run `yarn` again. Let me know if you have any issues.

_(This PR was recreated to fix a commit signing issue.)_
